src/runtime_src/core/common/api/xrt_device.cpp

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright (C) 2020-2022 Xilinx, Inc. All rights reserved.
-// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+// Copyright (C) 2020-2023 Xilinx, Inc. All rights reserved.
```

Comment:
Let's not update Xilinx Copyright (C), just do the AMD one.  Or am I missing something?